tests/parse.rs

```diff
@@ -47,7 +47,7 @@ fn parse_char() {
 #[test]
 fn parse_label() {
   parse_test_fixture(
-    "Hello, world",
+    "Hello world",
```

Comment:
Or should labels be allowed to include ',' ? Kinda funky
Thought it was an open and shut case as you split the list in fizzbuzz with ',' but now you implemented lists with only space :shrug:

Reply:
I didn't want to require `,` for lists, since that complicates the parsing, and didn't see a nice way to handle it so I just didn't to begin with. This test more documented that behaviour than declared a set-in-stone decision.

I really like the solution you have with treating `,` as whitespace, making it optional but usable as a separator, so consider me converted to that.
However, you should keep the `,` in the input string and remove it in the DataValue::Label below to verify that behaviour ;)